app/controllers/MonthlyContributions.scala

```diff
@@ -51,7 +54,7 @@ class MonthlyContributions(
       allowMembershipMail = true,
```

Comment:
That's going to have to be populated from an initially-unchecked checkbox shown to the user under GDPR :checkered_flag: